app/src/main/java/com/framgia/forder/screen/cart/notecart/NoteCartViewModel.java

```diff
@@ -0,0 +1,49 @@
+package com.framgia.forder.screen.cart.notecart;
+
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+import android.view.View;
```

Comment:
ERROR: Unused import: android.view.View. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports